### PR TITLE
Changes to support for 25.1 revision

### PR DESCRIPTION
**Pull request by guptadivya1501**

### CODE REVIEW

**Comment by PProfizi** [comment]
Hi @guptadivya1501, I think the changes required to support `25.1` were already made.

**Comment by guptadivya1501** [comment]
@PProfizi , If you check, it is merged yet.
image

File: src/ansys/dpf/core/_version.py [hunk 1]
```diff
@@ -23,7 +25,7 @@ class ServerToAnsysVersion:
         "7.1": "2024R1",
         "8.0": "2024R2",
         "8.1": "2024R2",
-        "8.2": "2024R2",
```
Comment (BClappe) [inline src/ansys/dpf/core/_version.py:26]: Why do we remove this line? @guptadivya1501

File: src/ansys/dpf/core/_version.py [hunk 2]
```diff
@@ -23,7 +25,7 @@ class ServerToAnsysVersion:
...
+        "9.0": "2025R1",
```
Comment (PProfizi) [inline src/ansys/dpf/core/_version.py:28]:
```suggestion
        "8.2": "2024R2",
        "9.0": "2025R1",
```

File: src/ansys/dpf/core/_version.py (hunk 1 above)
Comment (guptadivya1501) [reply src/ansys/dpf/core/_version.py:26]: @BClappe , Oops I dont know how it is removed. I will add again

**Review by PProfizi — APPROVED** [review]

**Pull request merged**